Development practices/Emails.md

```diff
@@ -0,0 +1,65 @@
+# Guidelines
+
+## Templates
+In most cases we don't need a custom design for the emails. We use [simple HTML templates](https://github.com/mailgun/transactional-email-templates) and modify them slightly with project's theme colors, and add a logo if needed.
+
+If there's a requirement for having a completely custom template consult with someone from the JS team on which libraries to use.
```

Comment:
Is there anything else in use except MJML? I knew it before, but the frontend developers didn't on a recent project, so it might make sense to list the most used options here so we know what to choose from.

Reply:
I'll mention MJML.